.rubocop.yml

```diff
@@ -130,6 +130,11 @@ Style/FrozenStringLiteralComment:
     - 'actionpack/test/**/*.ruby'
     - 'activestorage/db/migrate/**/*.rb'
 
+Style/RedundantFreeze:
+  Enabled: true
+  Exclude:
+    - 'activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb'
```

Comment:
Just excluded this file to avoid false positive until the fix made by https://github.com/rubocop-hq/rubocop/pull/6333 is available as the newer version of RuboCop, 0.60 maybe and Code Climate supports the new version of RuboCop to avoid auto correct by the false positive.